styles/style.css

```diff
@@ -150,6 +150,64 @@
     width: 20%;
 } 
 
+/* ****************** game in progress ************** */
+.answers {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    /* border: 2px solid wheat; */
```

Comment:
komentarz?

Reply:
Ustawiam tymczasowy border do wraperow/kontenerow w celu latwiejszego stylowania layoutow . Niestety czasem zapominam go usunac gdy juz nie jest potrzebny.

Reply:
oki, to usuń ;)